### DISCUSSION TITLE
Button Icon Color

### DISCUSSION THREAD

**Discussion by IbrahimSemab** [godotengine/godot-proposals#5948] [GUI]
I see that Godot has amazing Control nodes but I supposing if the color of icon of button could be changed.

**Comment by Zireael07** [comment]
Use the modulate property of the button

**Reply by Calinou** [reply]
Modulate property will also color the text, and it won't work as you expect with buttons that already have a dark background. Instead, you need to add StyleBox overrides to the `normal` style (and `hover`, `pressed`, etc).

**Reply by Zireael07** [reply]
You can color the text though with styles, too.
